### DISCUSSION TITLE
Docker version not checking me in–just keeps running

### DISCUSSION THREAD

**Discussion by davewadestein** [jdholtz/auto-southwest-check-in#108] [Q&A] [answered]
Howdy. I'm using a MacBook Pro (M1 chip) and getting SSL: CERTIFICATE_VERIFY_FAILED. I updated certifi, and that didn't solve the issue. So I've tried the Docker version and just before the 24-hour window I checked it. Still running, but no output. About 1-2 mins. into the 24-hour window I checked in for my flight manually. Then I deleted the container, relaunched it, and then I'd immediately see the "PASSENGER got BOARDING POSITION!" notice.

So it seems to not be checking me in, but after the fact, the script notices I am checked in.

Any thoughts on why it won't check in? My Mac is connected to the internet, shows the correct date and time, and is syncing with time.apple.com.

Thanks for any help!

P.S.Here's what it looks like in Docker Desktop while it's waiting. 
<img width="940" alt="Screenshot 2023-06-20 at 9 43 53 AM" src="https://github.com/jdholtz/auto-southwest-check-in/assets/15382758/b4de9883-cbd3-4916-9f2f-cbc111f65858">

**Comment by jdholtz** [comment, marked as answer]
While the container was running, did your Mac ever go to sleep? That would pause execution of the script, making the time longer than what it should be. 

Do you have the logs for the failed run from that container in `logs/auto-southwest-check-in.log` (if the container isn't fully deleted yet)? That would help to attach so we can further debug this.

**Reply by davewadestein** [reply]
Ah, for sure that was it. Newer computer and the settings were something I didn't pay attention to. Now I've set it to not sleep when the display dims and we'll see if that works...

<img width="465" alt="Screenshot 2023-06-20 at 8 02 32 PM" src="https://github.com/jdholtz/auto-southwest-check-in/assets/15382758/054d5f32-49ee-46db-b178-75df5c883cfa">
